compiler/cli/src/main/core/com/kitakkun/backintime/compiler/cli/BackInTimeCommandLineProcessor.kt

```diff
@@ -12,21 +12,30 @@ import org.jetbrains.kotlin.config.CompilerConfigurationKey
 class BackInTimeCommandLineProcessor : CommandLineProcessor {
     companion object {
         private const val KEY_ENABLED = "enabled"
+        private const val KEY_CONFIG_FILE = "config"
         val CONFIG_KEY_ENABLED = CompilerConfigurationKey.create<Boolean>(KEY_ENABLED)
+        val CONFIG_KEY_CONFIG_FILE = CompilerConfigurationKey.create<String>(KEY_CONFIG_FILE)
     }
 
-    override val pluginId: String = ""
+    override val pluginId: String = "com.kitakkun.backintime.compiler"
```

Comment:
note: If it is missing pluginId, the command line processor can't receive any options configured via Gradle Plugin.